include/ffconf.h

```diff
@@ -189,7 +189,11 @@
 
 
 #define	_MIN_SS		512
+#if defined(CONFIG_FS_FATFS_MAX_SS)
+#define	_MAX_SS		CONFIG_FS_FATFS_MAX_SS

```

Comment:
Update the README list of exported config parameters, similar to the PR; https://github.com/zephyrproject-rtos/fatfs/pull/4

Reply:
Updated README.